tribe-ext-settings-import-export.php

```diff
@@ -0,0 +1,370 @@
+<?php
+/**
+ * Plugin Name:       The Events Calendar Extension: Settings Import / Export
+ * Plugin URI:        https://theeventscalendar.com/extensions/settings-import-export/
+ * GitHub Plugin URI: https://github.com/mt-support/tribe-ext-settings-import-export
+ * Description:       You can import and export the settings of The Events Calendar
+ * Version:           1.0.0
+ * Extension Class:   Tribe\Extensions\Settings_Import_Export\Main
+ * Author:            Modern Tribe, Inc.
+ * Author URI:        http://m.tri.be/1971
+ * License:           GPL version 3 or any later version
+ * License URI:       https://www.gnu.org/licenses/gpl-3.0.html
+ * Text Domain:       tribe-ext-settings-import-export
+ *
+ *     This plugin is free software: you can redistribute it and/or modify
+ *     it under the terms of the GNU General Public License as published by
+ *     the Free Software Foundation, either version 3 of the License, or
+ *     any later version.
+ *
+ *     This plugin is distributed in the hope that it will be useful,
+ *     but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *     MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *     GNU General Public License for more details.
+ */
+
+namespace Tribe\Extensions\Settings_Import_Export;
+
+use Tribe__Autoloader;
```

Comment:
I _suspect_ this is from the extension template, but in this case we don't need or use autoloading and it can be removed.